api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/PatientTranslatorImpl.java

```diff
@@ -154,7 +154,9 @@ public Patient toFhirResource(@Nonnull org.openmrs.Patient openmrsPatient) {
 		}
 		
 		for (Identifier identifier : patient.getIdentifier()) {
-			currentPatient.addIdentifier(identifierTranslator.toOpenmrsType(identifier));
```

Comment:
If there a bug raised by calling `addIdentifier(null)`? I don't particularly like having to run the translation twice.

The other alternative is something like:

```java
PatientIdentifier omrsIdentifier = identifierTranslator.toOpenmrsType(identifer);
if (omrsIdentifier != null) {
    currentPatient.addIdentifier(omrsIdentifier);
}
```